pondi_planner/src/components/explorenav/explorenav.css

```
@@ -0,0 +1,50 @@
+body{
+	background: url("/Users/ashishsahni/Desktop/explorenav/Polygon 1.png") center / cover no-repeat;
```

The image path is wrong. Use the path which is accessible 
 i.e., in the code/github probably `assets/Polygon 1.png`

```suggestion
	background: url("assets/Polygon 1.png") center / cover no-repeat;
```